libmscore/score.cpp

```diff
@@ -273,31 +273,7 @@ Score::Score(MasterScore* parent, bool forcePartStyle /* = true */)
             _style = *MScore::defaultStyleForParts();
       else {
             // inherit most style settings from parent
-            _style = parent->style();
-
-            static const Sid styles[] = {
-                  Sid::pageWidth,
-                  Sid::pageHeight,
-                  Sid::pagePrintableWidth,
-                  Sid::pageEvenLeftMargin,
-                  Sid::pageOddLeftMargin,
-                  Sid::pageEvenTopMargin,
-                  Sid::pageEvenBottomMargin,
-                  Sid::pageOddTopMargin,
-                  Sid::pageOddBottomMargin,
-                  Sid::pageTwosided,
-                  Sid::spatium
-                  };
-            // but borrow defaultStyle page layout settings
-            for (auto i : styles)
-                  _style.set(i, MScore::defaultStyle().value(i));
-            // and force some style settings that just make sense for parts
-            if (forcePartStyle) {
-                  style().set(Sid::concertPitch, false);
-                  style().set(Sid::createMultiMeasureRests, true);
-                  style().set(Sid::dividerLeft, false);
-                  style().set(Sid::dividerRight, false);
-                  }
```

Comment:
Won't this break the things this code was designed to do - make sure that when you *create* parts, they inherit most score settings but override those four specific ones?